fmriprep/workflows/bold/tests/test_util.py

```diff
@@ -58,7 +58,8 @@ def symmetric_overlap(img1, img2):
     )
 ])
 def test_masking(input_fname, expected_fname):
-    bold_reference_wf = init_bold_reference_wf(omp_nthreads=1)
+    bold_reference_wf = init_bold_reference_wf(
+        omp_nthreads=1, remove_rotation_and_shear=False)
```

Comment:
Since you're not setting this to `False` in the actual pipeline, I would not set it `False` here. It's not obvious that a change to the algorithm that causes a 2% shift without this step will cause the same 2% shift with the step.

Reply:
AFNI ignores those when calculating transformations. If you keep them in the reference and use ANTs to apply the transformations estimated using AFNI the results will be misaligned.

Reply:
That makes sense. Do you think it would be possible to include rotation and shear [here in the `Volreg2ITK` conversion](https://github.com/poldracklab/fmriprep/pull/1354/files#diff-e5bf7060b352a39b7fdadceab77399d6R91)?

Reply:
Potentially, but that would be harder to implement.

Reply:
> What if we decide to fall back into mcflirt? Then this test would require rolling data back again.

As long as we always apply this cleaning to the entire BOLD series and , MCFLIRT will also be working from the cardinal RAS+ orientation. This would actually end up finishing up #260.